python/py-MDAnalysis/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-MDAnalysis
+version             0.19.2
+revision            0
+categories-append   science
+platforms           darwin
+license             GPL-2.0+
+
+python.versions     27 36 37
+
+maintainers         {gmail.com:giovanni.bussi @GiovanniBussi} openmaintainer
+
+description         An object-oriented toolkit to analyze molecular dynamics trajectories generated by CHARMM, Gromacs, NAMD, LAMMPS, or Amber.
+long_description    ${description} \
+                    It also reads other formats (e.g. PDB files and XYZ format trajectories. \
+                    It can write most of the coordinate formats, too, together with atom selections for use \
+                    in Gromacs, CHARMM, VMD and PyMOL.
+
+homepage            https://www.mdanalysis.org/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  7dab230b0e9770eeb0f760310660fe26dd70d9b2 \
+                    sha256  c5395bbafa5efca2e1aee4715d26129844140c47cb8301da0293106cb969de7d \
+                    size    16457970
+
+if {${name} ne ${subport}} {
+    build.env-append     MDA_USE_OPENMP=0
```

Comment:
whitespacing is incorrect here (as @mfk mentioned before); the right column should start at a multiple of 4 space (so not at 26, but either at 25 where you wrap the depends_build-append line or at 29).